tests/Doctrine/Tests/DBAL/Types/ArrayTest.php

```diff
@@ -7,9 +7,15 @@
 
 class ArrayTest extends \Doctrine\Tests\DbalTestCase
 {
-    protected
-        $_platform,
-        $_type;
+    /**
+     * @var MockPlatform
```

Comment:
The `MockPlatform` doesn't define any new API which is not defined in `AbstractPlatform`. Could we declare this property as `AbstractPlatform` and consider the mock an implementation detail?

Reply:
@morozov What do you suggest?

Reply:
@carusogabriel @morozov is suggesting to simply document this as `AbstractPlatform`

Reply:
@Ocramius Done!